tests/test_check_dagger.py

```
@@ -0,0 +1,327 @@
+def test_validate_runtime_image(cmd: str):
+    LOGGER.info("# Running tests for $(NOTEBOOK_NAME) runtime...")
+    #     # run_kubectl(["wait", "--for=condition=ready", "pod", "runtime-pod", "--timeout=300s"])
+    #     # LOGGER.error("Usage: make validate-runtime-image image=<container-image-name>")
+    #     # fail = False
+    image = "ghcr.io/jiridanek/notebooks/workbench-images:runtime-minimal-ubi9-python-3.9-jd_helpful_error_751147cd93fed327e940670edbc99c6f44a1ac24"
+    #     async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
+    container = testcontainers.core.container.DockerContainer(image)
+    container.with_command("tail -f /dev/null")
+    with container.start():
+        #         c = (client.container().from_(image))
+        for cmd in REQUIRED_RUNTIME_IMAGE_COMMANDS:
```

Pylint (3.3.7)

[refactor] Redefining argument with the local name 'cmd'

(R1704)